targets/ubuntu-14.04-enterprise/Dockerfile

```diff
@@ -115,3 +97,7 @@ RUN echo $IMAGE_TAG > /opt/circleci/image_version
 ADD targets/ubuntu-14.04-enterprise/pkg-versions.sh /opt/circleci/bin/pkg-versions.sh
 
 LABEL circleci.user="ubuntu"
+
+USER ubuntu
+
+CMD ["/bin/bash"]
```

Comment:
Ah, so I should not have added the CMD line? Where did George mention that this broke things?